muhozi/Weconnect-front-end
# #157552164 Set up testing environment

#### What does this PR do?
* Set testing environment and Write some Components tests
* Set up linting
* Refactor source codes to fix linting errors
* Set up Travis configuration and coverage reporting
* Add travis, coverage and codacy badges in the README

#### How should this be manually tested?
* Badges in the README

After setting app as the README says you can:
* Run `yarn test` for running tests
* Run `yarn lint` to check for linting errors
#### What are the relevant pivotal tracker stories?
#157552164
